src/components/TypesOfRecipeCards/RecipeCard.jsx

```diff
@@ -0,0 +1,91 @@
+import { PushPin } from "@phosphor-icons/react";
+import { Link } from "react-router-dom";
+import { pinWhite, pinSize } from "../../shared/style";
+
+//RecipeCard is now able to take in recipe and review, and render the correct information
+//Things updated
+// click an it navigates to recipe
```

Comment:
These comments don't really add anything and add more confusion to me. 

Good comments clarify what a piece of code does and why it's necessary, offering context and intent. Explain the “What” and “Why” not the “How”. Ideally, your code should be self-explanatory that if doesn't need a comment.

Reply:
https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/